.github/actions/report-run-failure/README.md

```diff
@@ -0,0 +1,30 @@
+## Notes on using this action
+
+  - GOVUK_SLACK_WEBHOOK_URL should be available as a secret in your repository, if it is not please post a message on #govuk-platform-engineering
```

Comment:
I would make this note more informative and remove the "please ask..." bit. It implies that we are expecting this not to work properly which shouldn't be the case. Recent user research confirmed product developers know how to reach our team. This repo is owned by us and the single source of truth of for repos ownership are the docs.

```suggestion
  - This action relies on GOVUK_SLACK_WEBHOOK_URL actions secret being configured for the repository. It’s added to all GOV.UK repositories as an organisation secret in the [GOV.UK GitHub Infrastructure configuration
](https://github.com/alphagov/govuk-infrastructure/blob/main/terraform/deployments/github/main.tf)
```

Reply:
ok, I'll pick up the wording